File: LTP/Exercicis practiques 1 i 2/Poliedro/Circulo.java
public class Circulo extends Figura {
    protected double r;

    public Circulo (double x, double y, double radio) {
        super(x,y);
        r = radio;
    }   
    
    public double area () {
        return Math.PI * Math.pow(r,2);
    }
}

File: FSO/PL02/FSO pract2/circulo.c
#include <stdio.h>
#include <area.h>
#define PI 3.1416 //mejor con const

int main() {   
   float area, radio;
   radio = 10;   
   area = PI * (radio * radio);   
   printf("Area del circulo de radio %f es %f\n", radio, area); 
}

